CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## Draft
+- Payment methods are not centered in the Cart page and Cart preview form. [#2072](https://github.com/bigcommerce/cornerstone/pull/2072)
```

Comment:
id of your pr - 2073, but not 2072

Reply:
I saw this after I created the PR, I have already fixed it, thanks!